Source/AssetRipper.GUI.Web/Pages/Assets/FontTab.cs

```diff
@@ -7,41 +7,68 @@ internal sealed class FontTab : HtmlTab
 {
 	public byte[] Data { get; }
 
+	public string Source { get; }
+
 	public string? FileName { get; }
 
+	public string? MimeType { get; }
+
 	public override string DisplayName => Localization.AssetTabFont;
 
 	public override string HtmlName => "font";
 
-	public override bool Enabled => Data.Length > 0;
+	public override bool Enabled => !string.IsNullOrEmpty(Source);
```

Comment:
Why?

Reply:
I just want it keep the same structure as https://github.com/AssetRipper/AssetRipper/blob/6771232426dc4a38e621ccaf034d916e11a2d79b/Source/AssetRipper.GUI.Web/Pages/Assets/AudioTab.cs#L15

Reply:
My intuition says that the previous version is more future-proof. I foresee Source being removed in some way when this becomes asynchronous.